db/models.py

```diff
@@ -45,3 +50,65 @@ class MovieSession(models.Model):
 
     def __str__(self):
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class User(AbstractUser):
+
+    @classmethod
+    def create_user(cls):
+        pass
```

Comment:
In this way it works also
```suggestion
class User(AbstractUser):
        pass
```